pages/common/dash.md

```diff
@@ -7,22 +7,30 @@
 
 `dash`
 
-- Execute a command and then exit:
+- Execute a [c]ommand:
 
 `dash -c "{{command}}"`
 
 - Execute a script:
 
-`dash {{path/to/script.sh}}`
+`dash {{path/to/script.dash}}`
 
-- Run commands from a script, printing each command before executing it:
+- Check a script for syntax errors:
 
-`dash -x {{path/to/script.sh}}`
+`dash -n {{path/to/script.dash}}`
 
-- Execute commands from a script, stopping at the first error:
+- Execute a script while printing each command before executing it:
 
-`dash -e {{path/to/script.sh}}`
+`dash -x {{path/to/script.dash}}`
 
-- Read and execute commands from stdin:
+- Execute a script and stop at a first [e]rror:
+
+`dash -e {{path/to/script.dash}}`
+
+- Execute a command from [s]tdin:
```

Comment:
Just a single command? Or multiple commands?

Reply:
Several commands. I've fixed this for all specified shells above.